Paper.tex

```diff
@@ -137,7 +137,7 @@ \subsection{Value}
 
 \subsection{Which History?}
 
-Since the system is decentralised and all parties have an opportunity to create a new block on some older pre-existing block, the resultant structure is necessarily a tree of blocks. In order to form a consensus as to which path, from root (the genesis block) to leaf (the block containing the most recent transactions) through this tree structure, known as the blockchain, there must be an agreed-upon scheme. If there is ever a disagreement between nodes as to which root-to-leaf path down the block tree is the `best' blockchain, then a \textit{fork} occurs.
```

Comment:
That sentence in the middle boils down to: _"In order to form a consensus as to which path [...] there must be an agreed-upon scheme"_

I feel like there should be a verb in there, tied to _"path"_. I opted for _"is to be considered canonical"_, but then I had to move things around a little.